lua/default-config.lua

```diff
@@ -117,9 +143,24 @@ lvim.lang = {
       },
     },
   },
+  crystal = {
```

Comment:
Will this work without the corresponding ftplugin file ?

Reply:
we have the ftplugin file though?

Reply:
I'm blind, sorry :laughing: